tests/test_distributed/test_with_spark/test_spark_local.py

```diff
@@ -158,6 +159,47 @@ def reg_with_weight(
     )
 
 
+RegData = namedtuple("RegData", ("reg_df_train", "reg_df_test"))
+
+
+@pytest.fixture
+def reg_data(spark: SparkSession) -> Generator[RegData, None, None]:
+    X = np.array([[1.0, 2.0, 3.0], [0.0, 1.0, 5.5]])
+    y = np.array([0, 1])
+    reg1 = xgb.XGBRegressor()
+    reg1.fit(X, y)
+    predt0 = reg1.predict(X)
+    pred_contrib0: np.ndarray = pred_contribs(reg1, X, None, False)
+
+    # convert np array to pyspark dataframe
+    reg_df_train_data = [
+        (Vectors.dense(X[0, :]), int(y[0])),
+        (Vectors.sparse(3, {1: float(X[1, 1]), 2: float(X[1, 2])}), int(y[1])),
+    ]
+    cls_df_train = spark.createDataFrame(reg_df_train_data, ["features", "label"])
```

Comment:
`cls` usually mean classifier.

Reply:
> `cls` usually mean classifier.

Based on my experience, "cls" is usually used as an abbreviation for "class", while "clf" is used as an abbreviation for "classifier".

Reply:
no specific request here. it's just an arbitrary convention in this project.

Reply:
Thx, Done.